SPECS/systemd/systemd-bootstrap.spec

```diff
@@ -1,7 +1,7 @@
 Summary:        Bootstrap version of systemd. Workaround for systemd circular dependency.
 Name:           systemd-bootstrap
 Version:        250.3
-Release:        11%{?dist}
+Release:        12%{?dist}
```

Comment:
It might be worth entangling this and the regular `systemd.spec` files, so that we're reminded that updating one may require an update to the other. Similar to what we have with `cyrus-sasl` and `cyrus-sasl-bootstrap`.